### DISCUSSION TITLE
Question about WebviewInstallMode

### DISCUSSION THREAD

**Discussion by EHW-liao** [tauri-apps/tauri#7035] [Q&A] [answered]
Can the embedBootstrapper in WebviewInstallMode specify a URL address for download? I would like to download the webView2 installation package from the URL address we specified during user installation。

**Comment by FabianLars** [comment, marked as answer]
That's not possible, but your other question from earlier could be the solution. Just change the url in the wix and nsis templates and it should work.

**Reply by EHW-liao** [reply]
get it,thank you!
